bower.json

```diff
@@ -16,6 +16,16 @@
     "bootstrap": "~3",
     "owasp-password-strength-test": "~1.3.0"
   },
+  "overrides": {
```

Comment:
bootstrap doesn't have a bower.json with the 'main' components, so wiredep can't get this components.
this line just made wiredep get them.